tools/aequatus/aequatus.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="aequatus-generator" name="aequatus generator" version="0.1.1">
+    <description>Aquatus</description>
```

Comment:
`<description>` should contain information not already in the `<name>`, otherwise can just be omitted.